LiskHQ/lisk-hub
# Remove unused code for forging - Closes #505

### What was the problem?

Forging component is not longer used, references should be removed in src

### How did I fix it?

Remove any reference in src, and assure e2e and unit pass

### How to test it?

Try to access `http://localhost:8080/#/main/forging` and verify that app redirects user to `/main/dashboard`, and that there are no console errors. 

### Review checklist
- The PR solves #505
- All new code is covered with unit tests
- All new features are covered with e2e tests
- All new code follows best practices
